### ISSUE TITLE
How to provide custom CA certificates to be able to use fnm?

### ISSUE THREAD

**Issue by papb** [Schniz/fnm#580]
```
$ fnm ls-remote
error: HttpError: https://nodejs.org/dist/index.json: Connection Failed: invalid peer certificate contents: invalid peer certificate: UnknownIssuer
```

Is there an environment variable that I can point to `/etc/ssl/certs/ca-certificates.crt` and make this work? For example, there is `REQUESTS_CA_BUNDLE` for HTTPS requests from Python (more specifically, from the `requests` library in Python).

**Comment by Schniz** [comment]
Is that a new error? Maybe related to the ureq migration

**Comment by Schniz** [comment]
Or, you actually need a custom certification?

**Comment by devinsm** [comment]
Hi @Schniz!

First of all thanks for all your hard work creating and maintaining this project! It seems like a great tool.

I'm having a similar issue. I'm behind a corporate firewall. The corporate firewall decrypts and inspects SSL traffic. I'm not a networking expert, but the long and short of it is that the tools I use need to trust the root signing certificate as provided by the firewall, or bypass the firewall via an approved corporate proxy. I set `HTTPS_PROXY` to do the latter. You can set environment variables @papb described to do the former. As far as I can tell, `fnm` doesn't support either.

Are there any work arounds I can use? Are you open to a PR?

**Comment by Schniz** [comment]
I think that we stopped supporting it when wet migrated to ureq from reqwest. I will create a build that uses reqwest for you to test. What do you think about it?

**Comment by devinsm** [comment]
That sounds good to me! I'm happy to test it out.

**Comment by Schniz** [comment]
Great! I will try making one tomorrow. I will open a PR and send you a link to the actions tab to download the binary for your OS :)

**Comment by Schniz** [comment]
Hi! Can you try using the binaries from #586?

**Comment by Schniz** [comment]
Ping @devinsm =)

**Comment by Schniz** [comment]
merged in 3fc1184bcfbe5bcdd5fc062f96a87fcb72f8a25c, please reopen if it does not work :)

**Comment by papb** [comment]
Hello, sorry to take so long to reply here! Just wanted to mention that I tried again, in the same environment, with the latest version of fnm, and everything worked fine. Thank you very much for looking into it.
